osu.Game/Users/UserActivity.cs

```diff
@@ -25,36 +25,49 @@ public class ChoosingBeatmap : UserActivity
             public override string Status => "Choosing a beatmap";
         }
 
-        public class MultiplayerGame : UserActivity
-        {
-            public override string Status => "Playing with others";
-        }
-
-        public class Editing : UserActivity
+        public class InGame : UserActivity
```

Comment:
Isn't this suppsoed to be `abstract`?

Reply:
Code Style rules required me to make it `protected`. Unless that is ignorable.

Reply:
It's accessed outside, no? it can't be `protected`